04LessonFour/in-class-project/index.html

```diff
@@ -63,12 +58,10 @@
         </div>
 
         <div class="overview" align="justify">
```

Comment:
looking good so far, be careful not to use any attributes like "align" on here. ALL your styling should be in the CSS.